docker-compose.override.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
```

Comment:
This file is for debugging purposes, which is loaded by default by Docker Compose.